python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -485,12 +502,14 @@ class RandomForestRegressor(Base):
         concat_model_handle = concatenate_trees(deref(model_handles))
 
         concat_model_ptr = <size_t> concat_model_handle
```

Comment:
Since this is a pointer, we should cast to `uintptr_t`. Also we should type this variable as well.

```suggestion
        cdef uintptr_t concat_model_ptr = <uintptr_t> concat_model_handle
```